bundles/mapping/mapwfs2/service/UserStyleService.js

```diff
@@ -32,7 +32,7 @@ export class UserStyleService {
 
         const layerStyles = this.styles.get(layerId);
```

Comment:
Use `this.getUserStylesForLayer(layerId)` and remove the following `if (layerStyles) {` block.